uiowa/uiowa
# Upgrade node_revision_delete

Alternative to #6226
Resolves https://github.com/uiowa/uiowa/issues/6347
Relates to #6381 

_Note: some content types are configured with a different number of revisions to keep._

# To Test

- Pick a content type to test. They should all be updated and match their previous settings. Sync the site that has the content type. 
- There should be no issues with the database update or config import.
- Create some revisions of the content. Try manually deleting a revision.
- Create more than the limit and run cron to see what is cleaned up.
- There should still be a message that displays the current limit for the content type.
- Donation link of #6347 should be gone
- _Note the ability to delete prior revisions of the revision you are deleting is no longer a feature. Most of the drush commands have also been removed._
